torch/utils/data/dataset.py

```diff
@@ -252,6 +252,41 @@ def __len__(self):
             total += len(d)  # type: ignore
         return total
 
```

Comment:
not sure why lint didn't fail here,  but we used to let 2 newlines between class definitions